### DISCUSSION TITLE
Remember x,y position on screen of set top window on reopening

### DISCUSSION THREAD

**Discussion by Panostis** [WindowTop/WindowTop-App#207] [Ideas]
Hello, there. Thanks for the chance to make suggestions. I use the pro portable version of your software a lot and i just love it. In save window configuration for top window if you close the window and reopen, it remembers it was set as top window but not where exactly it was situated on the screen. How about if it was possible that the window would reopen as top window in the same exact position it was on the screen prior to closing? 
Thanks a lot!

**Comment by gileli121** [comment]
@Panostis 

Thanks for the suggestion.
I will look for reference how much it is requested in https://github.com/microsoft/PowerToys 

What if you use multiple monitors? 
What if you open the app in another monitor?
It will use x,y position inside that monitor?

**Reply by Panostis** [reply]
Yes, indeed. If you use one monitor, remember x,y position and maybe size of window too (? But Maybe size of window is not windotop's job to handle ?). Maybe someone resizes an app window and puts it somewhere on the screen. So remember the exact position and size of window at reopening. If you use two monitors, remember the exact same position (and maybe size) on whatever screen  the window was placed prior to closing. Thanks a lot.
